v2/support/demo_get_set_app_metadata/assets/iframe.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
+</head>
+<body>
+  <h3>Apps framework metadata test:</h3>
+  <label for="formName">Enter form name to save:</label>
+  <input style="width:250px" name="formName" id="formName" value="">
+  </br>
+  <input type="checkbox" id="myCheckbox">My checkbox</br>
```

Comment:
Maybe rename this to be closer to its functionality?
```suggestion
  <input type="checkbox" id="roleRestrictionCheckbox">Enable Role Restriction</br>
```